src/component/LogPages/LogPages.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import ForgetPwdPage from './ForgetPwdPage/ForgetPwdPage';
+import LoginPage from './LoginPage/LoginPage';
+import './LogPages.scss';
+import ResetPwdPage from './ResetPwdPage/ResetPwdPage';
+import SignUpPage from "./SignUpPage/SignUpPage";
+
+export default function LogPages() {
```

Comment:
a bit naming issue, it seems the word "log" should be "account" or "authentication"